libshaderc/src/shaderc.cc

```diff
@@ -107,8 +107,8 @@ std::mutex compile_mutex;  // Guards shaderc_compile_*.
 // case.
 class StageDeducer {
  public:
-  StageDeducer() : kind_(shaderc_glsl_infer_from_source){};
-  StageDeducer(shaderc_shader_kind kind) : kind_(kind){};
+  StageDeducer() : kind_(shaderc_glsl_infer_from_source), error_(false){};
+  StageDeducer(shaderc_shader_kind kind) : kind_(kind), error_(false){};
```

Comment:
Using a default argument is looking better, as you can be more DRY.
But Coder's choice again.